### ISSUE TITLE
Try to make this a progressive web app

### ISSUE THREAD

**Issue by isaikatofficial** [Saikat-98/World-Covid-Tracker#6]
Try to make this a progressive web app(PWA) by implementing service worker

**Comment by Amit366** [comment]
Means converting website into app. 
Is this the issue??

**Comment by isaikatofficial** [comment]
Yes a progressive web app means it's not a complete app but it has some native like functionality like default page, even if the network isn't present the user should not see the blank screen or error message it should have a default page....

If you can make this a native app you can do that also, I have opened another issue regarding that check that.

**Comment by chait04** [comment]
please assign me this issue... i love pwa

**Comment by Amit366** [comment]
Sir, I have cloned the repo but how to run the website ?

**Comment by isaikatofficial** [comment]
@chait04 Please work on offline pages. 

Lets work on this together. @chait04 @Amit366

**Comment by Amit366** [comment]
Sir, I have made the PR. Please review it.

**Comment by isaikatofficial** [comment]
@Amit366 I just need to review the code once, then I will merge the PR. Please give me sometime.

**Comment by Amit366** [comment]
OK Sir

**Comment by chait04** [comment]
@Amit366  send the img of your report .. generated by lighthouse

**Comment by Amit366** [comment]
@chait04 which img should I send?

**Comment by isaikatofficial** [comment]
@Amit366 the report generated by lighthouse audit from dev tools of chrome

**Comment by Amit366** [comment]
![Screenshot (17)](https://user-images.githubusercontent.com/60662775/103754394-4ddf5180-5032-11eb-8a19-50a11f950e5e.png)

**Comment by Amit366** [comment]
Sir I made some changes and the report changed to:
![Screenshot (18)](https://user-images.githubusercontent.com/60662775/103754825-e37ae100-5032-11eb-8cea-107cf2ebf8ad.png)

**Comment by Amit366** [comment]
So should I made another PR??

**Comment by isaikatofficial** [comment]
Yes do it I will merge it accordingly...

**Comment by himanshu007-creator** [comment]
Please assign me this issue as well

**Comment by isaikatofficial** [comment]
What ideas you have to improve?? @himanshu007-creator
